joomla/joomla-cms
# Fix front end XTD menus and contacts buttons

Pull Request for Issue #12309
### Summary of Changes
### Testing Instructions
- Install latests 3.7.x branch https://github.com/joomla/joomla-cms/archive/3.7.x.zip
- Enable all editor xtd plugins
- Go to frontend, edit an article or something and try to add a menu link and a contact link

Without patch you get an error

Apply this PR and retry, you should be able to select a menu and a contact respectively 

EDIT: PLEASE TRY inserting an article as well!!!
### Documentation Changes Required

None

@infograf768 check this one
